rotkehlchen/types.py

```diff
@@ -164,6 +169,23 @@ def get_chain_for_etherscan(self) -> Optional['ChainID']:
 
         return None
 
+    @classmethod
+    def chain_to_blockscout(cls, chain_id: 'ChainID') -> 'ExternalService':
+        if chain_id == ChainID.ETHEREUM:
+            return ExternalService.BLOCKSCOUT
```

Comment:
just need to confirm here: No change happens for ethereum blockscout as before in the enum value right? Otherwise we need DB upgrade I think

Reply:
no change so no action required